pages/login/index.vue

```diff
@@ -15,6 +16,7 @@
       <label for="password">
         <input
           id="inputPassword"
+          v-model="inputPassword"
```

Comment:
missed on .trim over here, passwords are one of the fields that should definitely get trimmed.

Reply:
Added the same in the latest commit.